admin/data/sql/migrations/090.do.add-bulk-permissions-pupil-table.js

```diff
@@ -0,0 +1,7 @@
+'use strict'
```

Comment:
why is there no allocation in the g sheet?

Reply:
Was about to allocate it. it's done.